exotica_core/include/exotica_core/kinematic_tree.h

```diff
@@ -145,9 +154,16 @@ class KinematicTree : public Uncopyable
     const Eigen::MatrixXd& GetJointLimits() const { return joint_limits_; }
     void SetJointLimitsLower(Eigen::VectorXdRefConst lower_in);
     void SetJointLimitsUpper(Eigen::VectorXdRefConst upper_in);
+    void SetJointLimitsVelocity(Eigen::VectorXdRefConst velocity_in);
+    void SetJointLimitsAcceleration(Eigen::VectorXdRefConst acceleration_in);
+    void SetFloatingBaseLimitsPosXYZEulerZYX(const std::vector<double>& lower, const std::vector<double>& upper, const std::vector<double>& velocity, const std::vector<double>& acceleration);
+    void SetPlanarBaseLimitsPosXYEulerZ(const std::vector<double>& lower, const std::vector<double>& upper, const std::vector<double>& velocity, const std::vector<double>& acceleration);
```

Comment:
Default vel/acc limits to empty vector: ```const std::vector<double>& acceleration = {}```
This will keep the API backwards compatible.

Reply:
I have made this change, but isn't the fact that the function is overloaded keeping the API backwards compatible?

Reply:
Yes, the over-loaded will take care of it. We will have ABI (not API) breakage either way.

Reply:
Also, those input arguments for velocity and acceleration should be simply `double`

Reply:
OK - I will not make the change as the overloaded function takes care of it